nixos/modules/services/web-apps/baget.nix

```diff
@@ -0,0 +1,104 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.baget;
+
+  defaultConfig = {
```

Comment:
might be good to have this as the descriptive default for `extraConfig`, perhaps with a not that user configuration will be merged into this instead of replacing it. is this the upstream default config, implicit or otherwise?

Reply:
Yeah, this is an upstream configuration -- I don't remember exactly, but I think I set only options that BaGet requires on launch, and took them from documentation or an example config file.